app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +89,11 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [interpolated(params)['response'] || 'Event Created', code]
+      if interpolated['response_headers'].presence
```

Comment:
Yea, that would be a good improvement. The code feels a bit weird because it's not "rails-y" to do things with an http response outside of a controller.